frontend/src/lib/components/chart/chart-types/radar-chart/RadarChart.svelte

```diff
@@ -13,8 +13,8 @@
 			size: number;
 		}>;
 	};
-	export let width = 1000;
-	export let height = 400;
+	export let width = 700;
+	export let height = 600;
```

Comment:
Nit: should not do changes like these in PRs about something completely different probably.